file-formats/uist/examples/z_aff_example_uist.uist.json

```
@@ -0,0 +1,18 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example of UIST Object for ABAP file formats",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "standard"
```

Since `"abapLanguageVersion": "standard"` is the default, we usually don't render it. Therefore, I would just remove the value from the example

```suggestion
    "originalLanguage": "en"
```